src/compiler/output-targets/dist-custom-elements/test/dist-custom-elements.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { addCustomElementInputs } from '../index';
+
+describe('dist-custom-elements', () => {
+  it('should add custom element inputs', () => {
+    const buildCtx: any = {
+      components: [
+        {
+          isPlain: true,
+          sourceFilePath: './foo/bar.tsx',
+          tagName: 'my-tag',
+        },
+      ],
+    };
+    const bundleOpts: any = {
+      inputs: {},
+      loader: {},
+    };
+    const outputTarget: any = {
```

Comment:
Note: This is included in the patch file in my first comment on this PR

I think we have enough here to make this a `OutputTargetDistCustomElements`, we just need to import the type
```ts
import type * as d from '@stencil/core/declarations';
```
Then:
```ts
    const outputTarget: d.OutputTargetDistCustomElements = {
```